### PR TITLE
Coveralls

### PR DESCRIPTION
**Pull request by joserc87**
Added coverall support to implement #8.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # pirant
 
 [![Build Status](https://travis-ci.org/aayush26/pirant.svg?branch=master)](https://travis-ci.org/aayush26/pirant)
+[![Coverage Status](https://coveralls.io/repos/github/aayush26/pirant/badge.svg?branch=coveralls)](https://coveralls.io/github/aayush26/pirant?branch=coveralls)
```
Comment (aayush26) [inline README.md]: [Coverage Status](https://coveralls.io/github/aayush26/pirant?branch=master)

This above markdown I got from coveralls website

Comment (joserc87) [reply README.md]: Oh, of course, I forgot to change the branch to master in the URL!

**Comment by joserc87** [comment]
Now it should be better.

**Comment by aayush26** [comment]
Cool. Awesome work. Dude. Can you squash the 2 commits before I merge the PR?

**Comment by joserc87** [comment]
Commits squashed.

**Comment by aayush26** [comment]
[Doubt] coveralls build taking long time. Is it expected?
Url: https://coveralls.io/builds/13518583

**Comment by joserc87** [comment]
No. This is very weird. To me it takes about a minute. I don't know why it's taking so long. :/

**Comment by aayush26** [comment]
Can you share you coveralls build url?

**Comment by joserc87** [comment]
I just realized that my last coverall builds don't contain any data either. There must be something wrong. https://coveralls.io/repos/181854/builds.

**Comment by aayush26** [comment]
Ok. Your latest build does have coverage data. But prior to that, few builds didnt have. You can look and figure out what may have went wrong.

**Comment by joserc87** [comment]
Ok. I think I know what went wrong. Now it should work (fingers crossed). BTW, the commit is squashed already.

**Comment by aayush26** [comment]
Great. It worked perfectly. What went wrong?

**Comment by joserc87** [comment]
Ok, I have no idea what "Changes Unknown when pulling" is supposed to mean, but it's working. :)

**Comment by joserc87** [comment]
Before pushing the coverage to coveralls, I need to actually generate the report xD. It worked the first time because it saved that report. I changed the config file to do that before sending the data, so now it works.

**Comment by aayush26** [comment]
You got a rant for devrant, I guess. Unknown changes but working.. 😄 🤣 😝

**Comment by joserc87** [comment]
hahahaha true

**Comment by aayush26** [comment]
I was assuming that. Coverage data generation. While using nose in local, I ran exculsively ```nosetests --with-coverage --cover-package pirant``` to get coverage data

**Pull request merged**